blockchain/citizens/become-congressmen.md

```diff
@@ -77,43 +77,64 @@ To vote, you need to set the correct threshold and proposal. Different actions m
 
 ### Liberland legislation
 
-Congressmen are able to add and remove rights at the international treaty tier. The international treaty tier is represented as "tier=1". Learn more about these methods in `Thresholds`
+Congressmen are able to add and remove legislation at the international treaty tier. The international treaty tier is represented as "tier=1". Learn more about these methods in `Thresholds`.
+
+In Liberland, there are three tiers of international treaties: the "Constitution," "International Treaty," and "Law." These legal frameworks can only be altered by specific groups of individuals.
+
+- Constitution: This tier encompasses the rights outlined in the constitution. Any citizen can propose changes to this law through the "propose_rich_origin" mechanism.
```

Comment:
Citizens are expected to use our FE, which handles the `propose_rich_origin` vs `propose` automatically based on tier.